package.json

```diff
@@ -9,34 +9,32 @@
     "homepage": "https://github.com/bazelbuild/rules_nodejs",
     "license": "Apache-2.0",
     "devDependencies": {
-        "@bazel/bazel": "^0.21.0",
+        "//": "need release candidate of Bazel to pick up https://github.com/bazelbuild/bazel/pull/6980",
+        "@bazel/bazel": "^0.23.0-rc1b",
```

Comment:
Nice. That should fix with the rules_typescript pkg_tar issue I was having as well hopefully.